projects/swimlane/ngx-graph/src/lib/graph/graph.component.ts

```diff
@@ -912,8 +913,8 @@ export class GraphComponent extends BaseChartComponent implements OnInit, OnChan
    *
    */
   @HostListener('document:touchmove', ['$event'])
-  onTouchMove($event: any): void {
-    if (this.isPanning && this.panningEnabled) {
+  onTouchMove($event: TouchEvent): void {
```

Comment:
The `TouchEvent` type causes issues in some older versions of browsers, so we have to remove it.